pepper-and-carrot-ep6.msrt

```diff
@@ -186,7 +186,7 @@
 [rus] Ты даже додумался взять зелье, мою одежду и шляпу!
 [eng] You even thought to bring a potion, my clothes, and my hat!
 [jbo] .i do punji pa mixre .e ke ji'a le taxfu e le mapku
-[fra] Tu as même pensé à prendre une potion, mes vêtements, mon chapeau !
+[fra] Tu as même pensé à prendre une potion, mes vêtements, et mon chapeau!
```

Comment:
Grammatically speaking, "et" is correct but not compulsory. I had chosen not to include it for the sake of better lipsync (tested with an amateur voice actress)